libraries/OPAMP/src/OPAMP.cpp

```diff
@@ -0,0 +1,111 @@
+#include "OPAMP.h"
+#include <Arduino.h>
+
+/* Make sure this library fails to compile for unsupported boards. */
+#if !defined(ARDUINO_UNOWIFIR4) && !defined(ARDUINO_MINIMA)
+#error "Unsupported board for OPAMP library."

```

Comment:
Terrific :+1:

Reply:
Should I remove this `#error` alltogether? I specifically wanted the compilation to fail if someone selected the Portenta C33 board, for example.

Reply:
No, this is terrific!